packages/esm-patient-test-results-app/src/tablet-overlay/tablet-overlay.scss

```diff
@@ -0,0 +1,50 @@
+@import "~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/type/styles";
+@import "~carbon-components/src/globals/scss/vendor/@carbon/layout/scss/generated/spacing";
+@import "~@openmrs/esm-styleguide/src/vars";
+
+
+.tabletOverlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 9999;
```

Comment:
Try not to have completely unreasonable z-indexes. depending on the page 2, 10, or 11 is probably plenty. Compare this with the z-index of the side rail (which is currently also pretty unreasonable at 1000).